servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -2681,6 +2683,37 @@ public AuthHistoryDependencies getAuthHistoryDependencies(ResourceContext ctx, S
         return dbService.getAuthHistory(domainName);
     }
 
+    @Override
+    public void deleteExpiredMembers(ResourceContext ctx, Integer purgeResources) {
+        final String caller = ctx.getApiName();
+        logPrincipal(ctx);
+        validateRequest(ctx.request(), caller);
+
+        LOG.info("deleteExpiredMembers: purgeResources value: {}", purgeResources);
```

Comment:
this seems to be unnecessary since we already have the other info line below and if the value is invalid then the error response will log it as well.